### DISCUSSION TITLE
Question: how to email admin without an account?

### DISCUSSION THREAD

**Discussion by markcellus** [LemmyNet/lemmy-ui#1501] [General]
### Question

Hello, I'm having problems logging into an account I've created on a Lemmy instance.  I didn't signup with an email so I can't do "Forgot password". And I'm not on Mastodon and have no way of contacting the admins about it. Is there a default admin email address on the instance or something, so I can just email them about the issue? Thanks in advance

**Comment by krestenlaust** [comment]
Alright, I'll bite. Can I contact the admin on your behalf?
